qlik-sense-enterprise/azuredeploy.parameters.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
```

Comment:
for the hard-coded values here, just use defaultVaules in the template - keep just the GEN values.